config/locales/en.yml

```diff
@@ -223,13 +223,14 @@ en:
     copyright: "2020 Houdini Project"
   login:
     header: "Login"
-    message: "Welcome Back"
+    message: "Please Enter Login Information"
     email: "Email"
     password: "Password"
     logging_in: "Logging you in..."
     forgot_password: "Forgot Password?"
     get_started: "Get Started"
     enter_info: "Please enter your login information"
-    success: "Success"
+    success: "You have successfully signed in."
     errors: 
-      password_email: "Please enter a valid email address and password"
\ No newline at end of file
+      password_email: "Please enter a valid email address and password"
+      errorBoundary: "Something went wrong. Please reload the page."
```

Comment:
Keys in the locale file should be in snake_case, not camelCase. So this should be `error_boundary`.

Reply:
Sounds good. I will change this